embulk-core/src/main/java/org/embulk/spi/time/TimestampParser.java

```diff
@@ -91,7 +106,12 @@ public TimestampParser(ScriptingContainer jruby, String format, DateTimeZone def
 
     public TimestampParser(ScriptingContainer jruby, String format, DateTimeZone defaultTimeZone, String defaultDate)
     {
-        JRubyTimeParserHelperFactory helperFactory = (JRubyTimeParserHelperFactory) jruby.runScriptlet("Embulk::Java::TimeParserHelper::Factory.new");
+        // TODO get default current time from ExecTask.getExecTimestamp
+        Ruby runtime = jruby.getProvider().getRuntime();
```

Comment:
Do we have plans to eliminate this JRuby instance eventually (may not be in this PR)?